cypress/e2e/EngageSphere_GUI.cy.js

```diff
@@ -0,0 +1,57 @@
+// Renderiza o cabeçalho com um h1 e alternância de tema
+describe('Teste do cabeçalho e alternância de tema', () => {
+    beforeEach(() => {
+      cy.visit('/');
+    });
+
+    it('Deve renderizar o cabeçalho com um h1', () => {
+      cy.contains('h1', 'EngageSphere').should('be.visible');
+    });
+
+    it('Deve alternar o tema', () => {
+      cy.get('#theme-toggle-button').should('exist');
+      cy.get('body').should('have.attr', 'data-theme', 'light');
+      cy.get('#theme-toggle-button').click();
+      cy.get('body').should('have.attr', 'data-theme', 'dark');
+    });
+  });
+
+//Mostra a saudação padrão (ou seja, Hi there! ...)
+describe('Teste de saudação', () => {
+    it('Deve mostrar a saudação padrão "Hi there!"', () => {
+      cy.visit('http://localhost:3000/');
+      cy.contains('Hi there!').should('be.visible');
+    });
+  });
+
+
+//Volta para a lista de clientes ao clicar no botão "Voltar"
+
+describe('Teste de redirecionamento ao clicar no botão "Voltar"', () => {
+    beforeEach(() => {
+      cy.visit('http://localhost:3000/');
+    });
+
+    it('Deve redirecionar para a lista de clientes ao clicar no botão "Voltar"', () => {
+      cy.get('[data-testid=pagination]').should('exist');
+      cy.contains('Prev').should('be.visible').and('be.disabled');
+      cy.contains('Page 1 of').should('be.visible');
+      cy.contains('Next').should('be.visible').and('not.be.disabled');
```

Comment:
Que tal uma verificação positiva?

```suggestion
      cy.contains('Next').should('be.visible').and('be.enabled');
```